.palantir/revapi.yml

```diff
@@ -1018,6 +1018,43 @@ acceptedBreaks:
       old: "method void org.apache.iceberg.PositionDeletesTable.PositionDeletesBatchScan::<init>(org.apache.iceberg.Table,\
         \ org.apache.iceberg.Schema, org.apache.iceberg.TableScanContext)"
       justification: "Removing deprecated code"
+  "1.5.0":
+    org.apache.iceberg:iceberg-parquet:
```

Comment:
I don't think we can break existing APIs without deprecating them first. Please check https://iceberg.apache.org/contribute/#adding-new-functionality-without-breaking-apis

Reply:
@nastra thanks for the pointer. I have retained the existing versions of the methods but deprecated them.